src/Controller/Component/AuthComponent.php

```diff
@@ -597,7 +597,7 @@ public function deny($actions = null)
     public function setUser(array $user)
     {
         $this->session->renew();
-        $this->session->write($this->sessionKey, $user);
+        $this->session->write($this->sessionKey . '.user', $user);
```

Comment:
You are changing `U` to `u`. On purpose? in RC BC breaks should be avoided so close to the final release if possible.